### DISCUSSION TITLE
Share screen as a window and drawing on top of video & shared screen

### DISCUSSION THREAD

**Discussion by NicksonYap** [netless-io/flat#1802] [Ideas]
Hi Agora Flat team,

Thanks for the amazing open-sourced code
Greatly demonstrating Agora's SDK & API capabilites

Since drawing even works on PPT files & PDF files that are added to Whiteboard

I think it's amazing if drawings can be done on top of videos too

and, if we can add a "share screen window" on the whiteboard + drawing, then it would be an amazing addition

When collaborating on certain projects, it can be very helpful if a participant can share screen, and the other viewers can draw on top of it

Thanks!
Nickson

**Comment by TeacherRabbit123** [comment]
This would be awesome too. Currently, only Zoom allows annotation on a shared video screen. It would be a great addition to FLAT.

**Reply by jtxxviii** [reply]
@TeacherRabbit123 Master, there is no audio/sound during share screen. I tried to play something in youtube but no audio/sound. Thank you.

**Reply by hyrious** [reply]
@jtxxviii Flat does not have the audio track in sharing screen currently.

**Reply by Leooeloel** [reply]
Currently requires the screen sharer to play the sound externally to sync to others
